### PR TITLE
net-dialup/freeradius-client: Adding virtual/libcrypt as dependency

### PR DESCRIPTION
**Pull request by alarig**
For libcrypt.so

### CODE REVIEW

**Comment by DarthGandalf** [comment]
I believe 'sci-astronomy/stellarium: fix patches' commit doesn't belong to this PR

**Comment by alarig** [comment]
> I believe 'sci-astronomy/stellarium: fix patches' commit doesn't belong to this PR

I doesn’t, and I don’t understand at all why github included it…

**Comment by alarig** [comment]
I did git rebase -i and dropped the other commit, I hope it won’t cause issue while merging

File: net-dialup/freeradius-client/freeradius-client-1.1.7-r2.ebuild [hunk 1]
```diff
@@ -15,7 +15,10 @@ KEYWORDS="~alpha amd64 arm arm64 ~hppa ~ia64 ppc ppc64 ~riscv sparc x86"
 
 IUSE="scp shadow static-libs"
 
-DEPEND="!net-dialup/radiusclient-ng"
+DEPEND="
+	!net-dialup/radiusclient-ng
+	virtual/libcrypt
```
Comment (thesamesam) [inline net-dialup/freeradius-client/freeradius-client-1.1.7-r2.ebuild:20]: Needs subslot dep (:=), will fix on merging. Thank you!

https://wiki.gentoo.org/wiki/Project:Toolchain/libcrypt_implementation#Developer_information

**Pull request closed without merging**